### PR TITLE
Article 125: Euler Path

### PR DESCRIPTION
**Pull request by SimonDold**
#### Length of text ####
 :page_with_curl: short 

####  Difficulty to read ####
 :mouse:  light 

####  Difficulty to understand #### 
 :school_satchel:  easy 

####  Type of text #### 

 :bowtie:  informative 

####  Type of header #### 
 

 :boom:  catchy

### CODE REVIEW

**Comment by vogti** [comment]
The branch and the pull request are not named according to the schema.

**Comment by Teifun2** [comment]
**Plenum:**
- Mention connection House of ?Nicolas?
- Missing link to computer science

**Comment by SimonDold** [comment]
>Mention connection House of ?Nicolas?

i think the article would get too long with it (and it is not so well known worldwide only the german wikipedia article about Eulerian path mentions it and the wikipedia article about the house of nicolaus itself has only 5 languages, none of them are english).

>Missing link to computer science

Did this.

>Lesson Learned

i think explaining the properties a graph has to have to have an Eulerian path is too long and complicated for the article.

File: articles/article-125-euler-path.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<!-- BEGIN TITLE -->
+# How to navigate in Königsberg
+<!-- END TITLE -->
+
+<!-- BEGIN BODY -->
+The city of Königsberg is split by a river into 4 land masses. Those are connected by 7 lovely bridges that invite to a walk. The best route is of course the one which crosses each bridge exactly once.
```
Comment (noverney) [inline articles/article-125-euler-path.md]: when using numbers it is usually best practice to spell then out when they are greater then 100 (I have seen exceptions to this rule if it is part of a name like a street)

Comment (noverney) [reply articles/article-125-euler-path.md]: The city of Königsberg is split by a river into four separate land masses. They are connected by seven lovely bridges, which are pedestrian friendly.

Comment (noverney) [reply articles/article-125-euler-path.md]: The best route would be to cross each bridge exactly once. (I am correctly for something more succinct)

File: articles/article-125-euler-path.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+But what is this route? Leonard Euler showed that unfortunately no such route exists in Königsberg, thereby he introduced the world to graphtheory, a tool to study one of the most important structures in computer science.
```
Comment (noverney) [inline articles/article-125-euler-path.md]: Leonard Euler showed that unfortunately no such route exists in Königsberg, thereby introducing to the world graph theory, a tool to study one of the most important structures in computer science.

changed the sentence to not make it a run-on sentence which would have to be split in two.

**Pull request merged**